config/locales/bo/models.yml

```diff
@@ -67,8 +67,11 @@ bo:
         attributes:
             definition_subject_association:
                 branch:
-                    one: 'Branch'
-                    other: 'Branches'
+                    one: 'root subject'
+                    other: 'root subjects'
+                subsidiary:
```

Comment:
The attribute names are called branch and subject. So I would prefer that here we use the actual name of the attribute "subject" (and then for the actual content "subsidiary subject" which is how david wants to call that).

Reply:
addressed in cf10421